scripts/install.js

```diff
@@ -1,13 +1,14 @@
-var fs = require("fs-extra"),
-  path = require("path"),
-  archiver = require("archiver"),
-  child_process = require("child_process"),
-  CWD = process.cwd();
+import { readFileSync } from "node:fs";
+import fs from "fs-extra";
+import path from "node:path";
+import archiver from "archiver";
+import child_process from "node:child_process";
+const CWD = process.cwd();
 
 var localPath = (relativePath) =>
   relativePath ? path.resolve(CWD, relativePath) : path.resolve(CWD);
 
-var plugin = require(localPath("./js.json"));
+var plugin = JSON.parse(readFileSync(localPath("./js.json")));
```

Comment:
I changed this as a workaround for lack of ability to import json on ES modules; and the lack of a proper ecma standard for doing so (there is a weird `assert {type: "json" }` or `with {type: "json" }` proposal in the works for years, but it's not standardised and eslint doesn't support it.

However I am not super confident in it having the same portability given using of `fs`.